composer.json

```diff
@@ -1,22 +1,26 @@
 {
     "name": "pulse00/ffmpeg-bundle",
+    "type": "symfony-bundle",
     "description": "Symfony bundle to provide PHP-FFmpeg as a Symfony service (https://github.com/alchemy-fr/PHP-FFmpeg)",
-    "keywords": ["ffmpeg", "multimedia"],
+    "keywords": ["multimedia", "video processing", "video", "audio processing", "audio", "avconv", "ffmpeg", "avprobe", "ffprobe"],
     "license": "MIT",
-    "type": "symfony-bundle",
-    "require": {
-        "php-ffmpeg/php-ffmpeg": "0.5.*@dev"
-    },
-    "minimum-stability": "dev",
     "authors": [
         {
             "name": "Robert Gruendler",
             "email": "r.gruendler@gmail.com"
         }
     ],
-    "autoload": {
-        "psr-0": { "Dubture\\FFmpegBundle": "" }
+    "require": {
+        "php": "^5.3.9 || ^7.0",
+        "php-ffmpeg/php-ffmpeg": "^0.5 || ^0.6",
+        "symfony/dependency-injection": "^2.3 || ^3.0",
+        "symfony/framework-bundle": "^2.3 || ^3.0"
```

Comment:
Dependency Injection and framework bundle as dependency, because lazy attribute for services was introduced in 2.3 and otherwise `composer update --prefer-lowest` would fail.